lms/djangoapps/discussion/static/discussion/js/discussion_board_factory.js

```diff
@@ -58,6 +58,7 @@
                     collection: discussion,
                     course_settings: courseSettings,
                     mode: 'tab',
+                    discussionBoardView: discussionBoardView,
```

Comment:
you may want to move this line above to `mode: 'tab'`, just to keep consistency.